docs/tutorial/2-deploy-your-first-contract.md

```diff
@@ -13,18 +13,18 @@ This guide will walk you through deploying and interacting with contracts in REM
    By default, REMIX provides a sample workspace. Choose a Solidity contract to compile and click the **▶️** button.
    Alternatively, use the **Solidity Compiler** tab to compile the selected contract.
 
-   ![Compile contract](img/dapp_compile.jpeg)
+   ![Compile contract](img/dapp_compile.png)
 
 2. Deploy to Westend.
 
-   Select **Westend Testnet - MetaMask** environment in the **Deploy & Run** tab.
+   Select **Injected Provider - MetaMask** environment in the **Deploy & Run** tab.
    When prompted, allow REMIX to connect to MetaMask. Your account address and balance will be displayed under the **ACCOUNT** section.
```

Comment:
Should we mention here that insinde Metamask the Westend network from chapter 1 should be selected? And also that injected means it uses whatever is selected inside Metamask. It doesn't seem obvious to me.

Reply:
After adding to Metamask, it is selected automatically. I have added more context as requested.